hw/ip/flash_ctrl/lint/flash_ctrl.waiver

```diff
@@ -7,3 +7,16 @@
 # All leaf resets have a reset multiplex
 waive -rules TERMINAL_STATE -location {flash_ctrl_lcmgr.sv} -regexp {.*StInvalid.*} \
       -comment "StInvalid is intended to be a terminal state"
+
+# Remove errors from prim_* modules
```

Comment:
these are annoying... I think they pop up due to the `.*` use in the prim generator... it would be good if we can solve/waive these right in the generated prims at some point. can you maybe add a TODO comment that these waivers should be fixed in primgen and removed?

Reply:
added TODO comment